Package.resolved

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I'm not sure if it's good practice to include package resolutions with a library. We should be fetching the latest version of dependencies, to ensure that future versions don't break. This file prevents us from doing that.

Reply:
Added to .gitignore